### DISCUSSION TITLE
Clear search after select option

### DISCUSSION THREAD

**Discussion by acetinick** [orchidjs/tom-select#38] [Q&A] [answered]
Nice to have possibility to search again after item is added.  Currently have to delete manually current search text

Actual:
Need to clear key word to search another in the multiselect field

**Comment by oyejorge** [comment, marked as answer]
I could go either way on adding another configuration setting for this. 

In some situations, we need to keep the keyword (so users don't have to repeatedly type the same keyword for selecting multiple similar options).

[It is a fairly simple customization to clear the keyword](https://jsfiddle.net/q20dzath/) but adding another configuration setting could help clarify things.

**Comment by acetinick** [comment]
Ahh yea no need to have another option. This solution works great.

Thanks!
